test/index.test.js

```diff
@@ -17,23 +25,49 @@ const printTree = (nodes, indentLevel = 0) => {
 	));
 };
 
+const parseScript = (() => {
+	let parsed;
+	const prog = createProg(_parsed => parsed = _parsed);
+	return script => {
```

Comment:
had to do some hoops here, because `sade`'s `parse` is hot - it executes a command right away and I just have wanted to parse `argv`-like to options with `sade`'s registered rules so they can get passed down to imported `microbundle`